src/features/transfer/TransferTokenForm.tsx

```diff
@@ -430,11 +443,26 @@ function ReviewDetails({ visible, tokenRoutes }: { visible: boolean; tokenRoutes
           <div>
             <h4>{`Transaction${isApproveRequired ? ' 2' : ''}: Transfer Remote`}</h4>
             <div className="mt-1.5 ml-1.5 pl-2 border-l border-gray-300 space-y-1.5 text-xs">
-              <p>{`Remote Token: ${route?.destRouterAddress}`}</p>
+              <p className="flex">
+                <span className="min-w-[7rem]">Remote Token</span>
+                <span>{route?.destRouterAddress}</span>
+              </p>
               {isNft ? (
-                <p>{`Token ID: ${sendValue}`}</p>
+                <p className="flex">
+                  <span className="min-w-[7rem]">Token ID</span>
+                  <span>{sendValue}</span>
+                </p>
               ) : (
-                <p>{`Amount (${originUnitName}): ${sendValue}`}</p>
+                <>
+                  <p className="flex">
+                    <span className="min-w-[7rem]">{`Amount (${originUnitName})`}</span>
+                    <span>{sendValue}</span>
+                  </p>
+                  <p className="flex">
+                    <span className="min-w-[7rem]">{`Interchain Gas (${originUnitName})`}</span>
+                    <span>{igpQuote?.weiAmount || '0'}</span>
```

Comment:
Thoughts on formatting this based off the # of decimals? I think the "Amount" above is fair to have in wei because you already will have typed in the decimal-based amount to get to this point, but because people may not be familiar with interchain gas / what wei means, I wonder if this should be formatted

Reply:
I like wei for this because the relative number of digits give you a sense of the scale for gas fees. We could have both wei and units for both though